### ISSUE TITLE
saved courses

### ISSUE THREAD

**Issue by johnetegtmeier** [Courseplay/courseplay#1444]
Hi just wanted to say I LOVE this mod. and I wanted to let you know I experienced a problem... I tested it out in a test game I have going and when I started my regular game the course I saved in the test game showed up in my regular game. please don't take me wrong, I'm NOT complaining, I just wanted to let you know Keep up the Awesome work .... a true fan, John

**Comment by Satissis** [comment]
Hehe I'm guessing you haven't read the beta release issue we did :)
Courses are not linked to your savegame anymore, they are linked to the map, so even if you created a new savegame with the same map, you will have all the courses you already did for it.

This is so you don't have to rerecord all the routes each time you start an new save with the same map.

So it's not an issue but an feature we implemented, together with all courses are now saved in there own xml file, which making it to load faster, save faster and delete faster.
